keyboards/orthograph/info.json

```diff
@@ -0,0 +1,216 @@
+{
+    "keyboard_name": "Orthograph",
+    "url": "https://github.com/vladkvit",
+    "maintainer": "vladkvit",
+    "manufacturer": "vladkvit",
+    "build": {
+        "debounce_type": "asym_eager_defer_pk",
+        "lto": true
+    },
+    "debounce": 4,
+    "usb": {
+        "vid": "0x564B",
+        "pid": "0x0602",
+        "device_version": "1.0.0"
+    },
+    "features": {
+        "rgb_matrix": true,
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "development_board": "elite_c",
```

Comment:
@vladkvit 
I'm not seeing any Elite-C pins that aren't available with a Pro Micro defined here. Would you be able to confirm that this board can use (two) Pro Micro microcontroller?

Reply:
Pro Micro will not work due to height. I just updated the readme to be more clear.